docs/tutorials/enterprise_beginner/3_using_existing_blocks.md

```diff
@@ -342,7 +341,7 @@ With this done, you should be able to add a new Schedule block to the Front Page
 
 ### Set up overflow
 
-Now let's make use of the overflow functionality. In the settings of the Featured Articles block turn on overflow and select the Other Articles block as its overflow target. This controls how content will behave once it has to leave the first block. This is behavior you are surely familiar with from many websites: we have placed featured articles in the first Schedule block and planned the times on which they will be aired. When a new article appears in this block, the last article currently in it will be 'pushed off' and will land in the block designated as the overflow block – that means in the list of articles below. In this way the most current articles are showcased at the top, while older articles are still easily accessible from the front page.
+Now you will make use of the overflow functionality. In the settings of the Featured Articles block turn on overflow and select the Other Articles block as its overflow target. This controls how content will behave once it has to leave the first block. This is behavior you are surely familiar with from many websites: you have placed featured articles in the first Schedule block and planned the times on which they will be aired. When a new article appears in this block, the last article currently in it will be 'pushed off' and will land in the block designated as the overflow block – that means in the list of articles below. This way the most current articles are showcased at the top, while older articles are still easily accessible from the front page.
```

Comment:
General question here: where is the change?
Is it a setting of your software(s) that induces a change or non-visible chars?

Reply:
from: Now let's make use of the overflow functionality
to: Now you will make use of the overflow functionalit